app/src/main/res/values/strings.xml

```diff
@@ -604,6 +604,8 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_construction_steps_title">Are these steps completed?</string>
     <string name="quest_construction_generic_title">Is this way completed?</string>
     <string name="quest_construction_building_title">Is this building completed?</string>
+    <string name="quest_fuelSelfService_title">Can you pump your own gas at this fuel station?</string>
```

Comment:
Sounds like you can bring your own gas to pump here.

Reply:
```suggestion
    <string name="quest_fuelSelfService_title">Can you pump gas on your own at this fuel station?</string>
```

Reply:
Mh I don't know how to write it because English isn't my mother tongue 😕 I trust @riQQ 😅

This is a preview (I also changed the icon):


<img src="https://user-images.githubusercontent.com/63296731/149673728-3d0eb224-2374-41ec-8dc4-0b9f14fc9a5b.png" width="200px">

Reply:
I propose "Can you pump gas by yourself at this fuel station?".

Reply:
@timothywashere suggested something along the line of "Dispense your own fuel?"

Reply:
"Do you dispense your own fuel at .... ?"

"Do you operate the pump yourself at ...?"

In terms of absolute clarity I think maybe:
"Are the pumps at ... operated by the customer?"

Reply:
I also asked in OSM slack US channel #language: 

![image](https://user-images.githubusercontent.com/4661658/149805009-3c193738-9502-48b0-8b44-813d303932d6.png)

Reply:
Yes, undoubtably "Pump your own gas" is correct in American English.